src/util/edit.cc

```diff
@@ -139,9 +183,19 @@ void EditEngine::clear() {
   edit_stack_.clear();
 }
 
-uint64_t EditEngine::byteValue(size_t byte_pos) const {
-  const auto it = itFromPos(byte_pos);
+uint64_t EditEngine::byteValue(size_t pos) const {
+  auto next_it = address_mapping_.upperBound(pos);
+  assert(next_it != address_mapping_.cbegin());
+  auto it = next_it;
+  --it;
+
+  if (it->fragment_ == nullptr) {
+    return original_data_->binData().element64(it->offset_ + (pos - it.key()));
+  } else {
+    return it->fragment_->element64(it->offset_ + (pos - it.key()));
```

Comment:
extract duplicated `it->offset_ + (pos - it.key())` to a variable with a descriptive name

Reply:
Now in lines 136-139.

Reply:
DONE